### ISSUE TITLE
SAS class comment mentions inexistant path

### ISSUE THREAD

**Issue by MarcoGorelli** [pandas-dev/pandas#31391]
`pandas/asv_bench/benchmarks/io/sas.py` :

> class SAS:
> 
>     params = ["sas7bdat", "xport"]
>     param_names = ["format"]
> 
>     def setup(self, format):
>         # Read files that are located in 'pandas/io/tests/sas/data'

However, the files are in  'pandas/tests/io/sas/data'.

EDIT:
unrelated typo: there's lots of places in the codebase that use "rather then" instead of "rather than". As this is also a minor typo that doesn't affect any code, I think it'd be OK to address both in the same pull request

**Comment by jeandersonbc** [comment]
Hi @MarcoGorelli,
about the "rather then" typo, I found those occurrences:

```
$ grep -nri "rather then" ..
../doc/source/whatsnew/v0.13.0.rst:217:- default for `display.max_seq_len` is now 100 rather then `None`. This activates
../pandas/core/config_init.py:9:module is imported, register them here rather then in the module.
../pandas/io/formats/printing.py:77:# Try to use these as much as possible rather then rolling your own.
../pandas/io/formats/printing.py:101:    rather then calling this directly.
../pandas/io/formats/printing.py:136:    rather then calling this directly.
../pandas/tests/io/formats/test_format.py:299:        # for the case of Index, where the repr is traditional rather then
../pandas/tests/groupby/test_grouping.py:391:        # the surprise comes from using key[0:6] rather then str(key)[0:6]
```

Is it ok if I submitt a PR and assign you as the reviewer?

**Comment by MarcoGorelli** [comment]
Closed by #31395
